src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -12,6 +12,7 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <license type="expression">MIT</license>
     <projectUrl>https://github.com/microsoft/sarif-sdk</projectUrl>
+    <icon>triskelion.png</icon>
```

Comment:
>triskelion [](start = 10, length = 10)

more recent nuget clients expect an embedded icon rather than simply having a URL to one.